docs/01-start-here/01-installation-steps.md

```diff
@@ -191,6 +191,8 @@ Now check that you are up and running by hitting the following URLs:
 * [http://localhost:3000/media/2012/dec/05/newspaper-editors-sign-up-leveson](http://localhost:3000/media/2012/dec/05/newspaper-editors-sign-up-leveson)
 * [http://localhost:3000/news/gallery/2012/dec/04/24-hours-in-pictures-gallery](http://localhost:3000/news/gallery/2012/dec/04/24-hours-in-pictures-gallery)
 
+**Nb**: The webserver started by sbt runs on port 9000, therefore all above links would work when replacing port 3000 by 9000. The server at 3000 is ran by `make watch` which has the added advantage of monitoring CSS and JavaScript assets.  
```

Comment:
What about instead:

Please note, `make watch` proxies port 3000 with the ability to live reload assets, you could access all links using port 9000 as well.